.config-schema.json

```diff
@@ -15,7 +15,7 @@
         "maxConcurrentReadsForCheck": {
             "description": "The maximum allowed number of concurrent reads in a single Check query.",
             "type": "integer",
-            "default": 30,
+            "default": 4294967295,
```

Comment:
Can we add comment that this is MaxUint32?

Reply:
A comment in a JSON file? No :/

Reply:
Put it in the description line.

Reply:
I feared that this description got printed to customers when using `openfga run --help`, but no, that is taken from somewhere else :) added

Reply:
> A comment in a JSON file? No :/
I meant in the description e.g. `(defaults to max uint32)` 👍